docs/content/concepts/layer/implementation/ssp/_index.md

```diff
@@ -66,6 +66,46 @@ The figure below expresses represents the portion of the OSCAL stack as it relat
 
 {{<partialCached "note-to-developers-uuid.html" >}}
 
+### Identifier References
+
+An OSCAL SSP may contain references to information that is defined locally (e.g., in the SSP model) or externally (e.g., in a referenced profile, catalog, or component definition model).  The following lists the mechanisms through which objects in an OSCAL SSP may "link" to other OSCAL content:
```

Comment:
The locally defined references reside in OSCAL SSP content or Catalog/Profile/CDef content, and not in the mentioned models) .

Reply:
Use instance vs cross-instance terminology from the [identifier scoping](https://pages.nist.gov/OSCAL/concepts/identifier-use/#scope).

Reply:
Reworded as:
_An OSCAL SSP may contain references to information that is instance scoped (e.g., in the SSP model), cross-instance scoped (e.g., in a referenced profile, catalog, or component definition model), or external (e.g., references to non-OSCAL resources). The following summarizes the scenarios where an OSCAL SSP may have references to other content._